draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -1401,6 +1400,9 @@ version is used. Reception of a non-verifiable MP_HMAC ({{MP_HMAC}}) or an inval
 MP_JOIN Path Token {{MP_JOIN}} during flow establishment MUST cause the
 subflow to be closed.
 
+Similar procedure MUST be applied if a final ACK carrying MP_KEY with wrong Key-A/Key-B is
```

Comment:
Insert /A/ before similar. I am not sure it is good practice to say a similar procedure in an RFC2119 sentence without cross reference,

Reply:
Does https://github.com/markusa/ietf-multipath-dccp/pull/222/commits/a27dd87f8ddadb2eaa1f02acb386b441ae31d97a address your concerns?